package.json

```diff
@@ -101,7 +101,7 @@
 
     "protractor": "^3.2.2",
 
-    "parse5": "^1.5.1",
+    "parse5": "^2.1.5",
```

Comment:
why is that? i have been working with 2.1.5 in all my projects without any problems and without getting any errors/warnings about that